packages/npm-dependencies-validation/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@sap-devx/npm-dependencies-validation",
+  "version": "1.3.0",
```

Comment:
I think the mono repo is not at version `1.3.2` please align

Reply:
Done